joomla-projects/privacy-framework
# Fix issue 125

Pull Request for Issue #125 .

### Summary of Changes
This PR does two things:
1. Only show IP column in the email if IP Logging is enabled
2. Use Text to show IP column to avoid a language item displayed as reported in #125

### Testing Instructions
1. See https://github.com/joomla-projects/privacy-framework/issues/125 for the issue description
2. Apply patch, confirm that the issue is fixed
3. Check and confirm when IP logging is disabled, the IP column is hided in email
